chartfx-samples/src/main/java/de/gsi/chart/samples/ContourChartSample.java

```diff
@@ -347,7 +346,19 @@ public void set(final int xIndex, final int yIndex, final double x, final double
             xValues[xIndex] = x;
             yValues[yIndex] = y;
             zValues[xIndex][yIndex] = z;
+        }
 
+        @Override
+        public double get(int dimIndex, int index) {
+            switch (dimIndex) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/app/GSI/chart-fx/pullRequest?prid=4477772)